pkg/orchestration/wiring/datadog/types.go

```diff
@@ -0,0 +1,57 @@
+package datadog
+
+import (
+	"github.com/atlassian/voyager"
+)
+
+type AlarmSpecType string
+type AlarmType string
+
+const (
+	Metric AlarmSpecType = "metric alert"
+)
+const (
+	CPU    AlarmType = "cpu"
+	Memory AlarmType = "memory"
+)
+
+type Alarm struct {
+	Name    string      `json:"name"`
+	Type    string      `json:"type"`
+	Query   string      `json:"query"`
+	Option  AlarmOption `json:"options"`
+	Message string      `json:"message"`
+}
+
+type ServiceInstanceSpec struct {
+	ServiceName voyager.ServiceName `json:"serviceName"`
+	Attributes  Alarm               `json:"attributes"`
```

Comment:
either rename `Alarm` type to `AlarmAttributes`, or rename `Attributes` to `Alarm` (and in JSON as well), to make them semantically closer